doc/en/user/source/community/wfs-templating/configuration.rst

```diff
@@ -222,4 +299,76 @@ The evaluation of a filter is handled by the module in the following way:
   
   * in case of a static content attribute, the static content will be included in the output if the filter condition is true.
   
-  * in case the expression is not matched, the content, static or dynamic, will not be set, resulting in the attribute being skipped.
\ No newline at end of file
+  * in case the expression is not matched, the content, static or dynamic, will not be set, resulting in the attribute being skipped.
+
+
+Inspire GeoJSON Output
+----------------------
+
+In order to provide a GeoJSON output encoded following INSPIRE rule for `alternative feature GeoJSON encoding <https://github.com/INSPIRE-MIF/2017.2/blob/master/GeoJSON/ads/simple-addresses.md>`_, it is possible to provide a VendorOption in the template file by adding the following attribute :code:`"$VendorOptions": "flat_output:true"`.
```

Comment:
Indeed it looked like that, but it was a leftover, since json-ld was not supposed to support the flat_output param, I've cleaned up the code. An example for the GeoJSON INSPIRE output has been included in the output section of documentation https://github.com/geoserver/geoserver/pull/4344/commits/7e555b46c6394c0cd2de6db085f519c846f623b7#diff-32de36cbd8049c697cfbeba5baf98c9bR247